src/components/_internal/BaseButton/BaseButton.types.ts

```diff
@@ -29,9 +38,11 @@ export interface BaseButtonProps
   href?: string;
   to?: To;
   className?: string;
-  iconName?: SSCIcons | string;
+  iconStart?: IconStart;
+  iconEnd?: IconEnd;
+  iconName?: SSCIcons | string; // @deprecated Use iconStart property instead
+  iconType?: IconTypes | string; // @deprecated Use iconStart property instead
```

Comment:
I think this won't work as JSDoc comment, it should be like this:
```suggestion
  /** @deprecated Use iconStart property instead */ 
  iconName?: SSCIcons | string;
  /** @deprecated Use iconStart property instead */
  iconType?: IconTypes | string;
```